CsvParser/csvParser_test.go

```diff
@@ -54,13 +54,13 @@ func Test_CsvParserSize(t *testing.T) {
 		{
 			name:  "The size of the incidents array",
 			input: "incidentfile.csv",
-			want:  100,
+			want:  40,
```

Comment:
Maybe change the name of this test (even though this existed before) to something that is indicative of how the numIncidents parameter influences CsvParser.